src/interpreter/mod.rs

```diff
@@ -73,7 +78,7 @@ impl Into<String> for Error {
 		match self {
 			Error::Program(s) => s,
 			Error::Validation(s) => s,
-			Error::Initialization(s) => s,
+			Error::Instatiation(s) => s,
```

Comment:
instantiation?

Reply:
Yep, it is term from the [spec](https://webassembly.github.io/spec/core/intro/overview.html). It is a process when initialization the state of the module (like defined globals, memories, etc) and imports matching takes place. OTOH `Initialization` in this contex seemed too ambigious for me . So I decided to change it to more domain-specific term.

Reply:
i'm about typo :)

Reply:
😳oops